src/config/dynamic-theme-fixes.config

```diff
@@ -6503,6 +6505,23 @@ a[href*="about/products"]
 img[src$="menu_black_24dp.png"]
 #assistive-chips .e2moi img
 button[guidedhelpid="searchbutton"]
+img[src="https://mt.googleapis.com/vt/icon/name=icons/onion/1491-wht-line-blank.png&filter=ff000000"]
+div[role="checkbox"] > div
+div[aria-label="Toggle star"][role="checkbox"]
+div.uVccjd.HzV7m-pbTTYe-KoToPc-ornU0b-hFsbo.HzV7m-KoToPc-hFsbo-ornU0b
```

Comment:
Their are quite a lot of "nonsense" css selectors here, given these are google maps, they tend to change quite a lot.

Reply:
Are you referring to that last line in particular or overall?

Reply:
> Are you referring to that last line in particular or overall?

Overall.